eitprocessing/datahandling/eitdata.py

```diff
@@ -111,43 +105,14 @@ def _sliced_copy(
             vendor=self.vendor,
             time=time,
             framerate=self.framerate,
-            phases=phases,
-            events=events,
             label=self.label,  # newlabel gives errors
             pixel_impedance=pixel_impedance,
         )
 
     def __len__(self):
         return self.pixel_impedance.shape[0]
 
-    @property
-    def global_baseline(self) -> np.ndarray:
-        """Return the global baseline, i.e. the minimum pixel impedance across all pixels."""
-        return np.nanmin(self.pixel_impedance)
-
-    @property
-    def pixel_impedance_global_offset(self) -> np.ndarray:
-        """Return the pixel impedance with the global baseline removed.
-
-        In the resulting array the minimum impedance across all pixels is set to 0.
-        """
-        return self.pixel_impedance - self.global_baseline
-
-    @property
-    def pixel_baseline(self) -> np.ndarray:
-        """Return the lowest value in each individual pixel over time."""
-        return np.nanmin(self.pixel_impedance, axis=0)
-
-    @property
-    def pixel_impedance_individual_offset(self) -> np.ndarray:
-        """Return the pixel impedance with the baseline of each individual pixel removed.
-
-        Each pixel in the resulting array has a minimum value of 0.
-        """
-        return self.pixel_impedance - self.pixel_baseline
```

Comment:
We decided a while ago to remove these. The naming is not very intuitive. Also, we wanted to have these values saved as ContinuousData rather than re-calculate them each time they are needed.